examples/client-logger/app-insights-logger/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const path = require("path");
+
+const webpack = require("webpack");
+const { merge } = require("webpack-merge");
+
+const fluidRoute = require("@fluid-tools/webpack-fluid-loader");
+
+const testAppSrcPath = path.join(__dirname, "src", "app");
+
+module.exports = (env) => {
+	const isProduction = env && env.production;
+
+	return merge(
+		{
+			entry: {
+				main: path.join(testAppSrcPath, "index.tsx"),
+			},
+			resolve: {
+				extensions: [".ts", ".tsx", ".js"],
+			},
+			module: {
+				rules: [
+					{
+						test: /\.tsx?$/,
+						loader: require.resolve("ts-loader"),
+					},
+					{
+						test: /\.css$/,
+						use: [
+							require.resolve("css-loader"), // translates CSS into CommonJS
+							require.resolve("style-loader"), // creates style nodes from JS strings
+						],
+					},
+				],
+			},
+			output: {
+				filename: "[name].bundle.js",
+				path: path.resolve(__dirname, "dist"),
+				library: "[name]",
+				devtoolNamespace: "fluid-tools/client-debugger-view",
+				libraryTarget: "umd",
+			},
+			plugins: [
+				new webpack.ProvidePlugin({
+					process: require.resolve("process/browser"),
```

Comment:
@alexvy86 has a PR up for client-devtools-view removing this. I suspect we don't need this here either?